### DISCUSSION TITLE
Need to run openssl as a daemon in node-red addon

### DISCUSSION THREAD

**Discussion by smwoodward** [hassio-addons/addon-node-red#934] [Q&A] [answered]
Title sums it up really, I need to be able to run openssl as a daemon inside the node-red container, and I'm not able to figure out how to do that yet.  Can anyone help me out?

**Comment by frenck** [comment, marked as answer]
Pretty simple. You can't.

**Reply by smwoodward** [reply]
Is there another application that I could possibly use?  I've been using the command in another node-red instance "openssl s_client xxx.xxx.xxx.xxx -port 12345 that has worked for being able to pass commands back and forth between my alarm panel and node red.  @frenck

**Reply by frenck** [reply]
No that I'm aware of. Maybe a Home Assistant integration you could use instead? Or a add-on available by someone?

My knowledge on this end with; this doesn't work with this add-on.
